webrender_api/src/api.rs

```diff
@@ -40,9 +40,7 @@ pub enum ResourceUpdate {
 
 impl ResourceUpdates {
     pub fn new() -> Self {
-        ResourceUpdates {
-            updates: Vec::new(),
-        }
+        Self::default()
```

Comment:
While technically more compact, this really just seems like it's making the code more opaque and harder to change later, especially since we have no use for the actual Default impl otherwise. I'd rather not land this particular change.